### PR TITLE
rustdoc: reorg to frontend subteam for UI/UX and main team

### PR DESCRIPTION
**Pull request by notriddle**
https://rust-lang.zulipchat.com/#narrow/stream/393423-t-rustdoc.2Fmeetings/topic/meeting.202024-06-10/near/438892399

### CODE REVIEW

**Comment by GuillaumeGomez** [comment]
Thanks! Let's ping the whole team to see if others want to be part of it as well.

cc @rust-lang/rustdoc

**Review by fmease — APPROVED** [review]

**Review by Manishearth — APPROVED** [review]

**Comment by fmease** [comment]
I think the PR's title meant to mention the team rustdoc-contributors not rustdoc?

File: teams/rustdoc-frontend.toml [hunk 1]
```diff
@@ -17,6 +20,12 @@ orgs = ["rust-lang", "rust-lang-nursery"]
 extra-teams = ["rustdoc"]
 
 [website]
-name = "Rustdoc team contributors"
-description = "Contributing to Rustdoc on a regular basis"
+name = "Rustdoc web frontend"
+description = "Rustdoc frontend design and development"
```
Comment (fmease) [inline teams/rustdoc-frontend.toml:24]: Should we mention the terms UI, UX and/or HTML somewhere here? Or just "frontend" to "web frontend" to disambiguate it?

/me having a giggle about the fact that rustdoc's (web) frontend is actually one of rustdoc's backends from a compiler perspective.

Comment (aDotInTheVoid) [reply teams/rustdoc-frontend.toml:24]: just wait till rustdoc has a seperate team responsible for cleaning from HIR and metadata :P

**Review by aDotInTheVoid — APPROVED** [review]
yes, please!

**Review by camelid — APPROVED** [review]
I would also like to be part of the new team.

File: teams/rustdoc-frontend.toml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+[website]
+name = "Rustdoc web frontend"
+description = "Rustdoc frontend design, UI, UX and development"
+zulip-stream = "t-rustdoc"
+
+[rfcbot]
+label = "T-rustdoc-frontend"
+name = "Rustdoc frontend"
+ping = "rust-lang/rustdoc"
```
Comment (camelid) [inline teams/rustdoc-frontend.toml:30]: I think it should be this instead?

```suggestion
ping = "rust-lang/rustdoc-frontend"
```

**Comment by fmease** [comment]
> Should I change the title to "rustdoc-contributors: reorg to frontend subteam"? 

Ah, no it's fine as is :+1:. I think I misread the title

**Comment by jackh726** [comment]
Is this good to merge? r? @GuillaumeGomez

File: teams/rustdoc-frontend.toml (hunk 2 above)
Comment (GuillaumeGomez) [reply teams/rustdoc-frontend.toml:31]: It should indeed!

**Comment by GuillaumeGomez** [comment]
Not yet, waiting for https://github.com/rust-lang/team/pull/1474/files#r1637422119 to be updated then it's good to go.

**Review by GuillaumeGomez — APPROVED** [review]

**Comment by GuillaumeGomez** [comment]
Thanks @notriddle !

@jackh726 It's now ready for merge.

**Review by jackh726 — APPROVED** [review]

**Pull request merged**